src/Umbraco.Core/Models/IContent.cs

```diff
@@ -77,6 +80,44 @@ public interface IContent : IContentBase
         /// </summary>
         ContentStatus Status { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether a given culture is available.
+        /// </summary>
+        /// <remarks>
+        /// <para>A culture becomes available whenever the content name for this culture is
+        /// non-null, and it becomes unavailable whenever the content name is null.</para>
+        /// </remarks>
+        bool IsCultureAvailable(int? languageId);

```

Comment:
Historically, implementation first depended on fields private to the content class. Not the case anymore, but have kept it here just in case. Wasn't entirely sure that relying on names was entirely OK. Suggesting we keep it as-is as per this PR and taking note in the other task (U4-11250) to consider moving to extension method. Is that OK?